app/assets/stylesheets/views/_landing_page/box.scss

```diff
@@ -0,0 +1,26 @@
+@import "govuk_publishing_components/individual_component_support";
+
+.app-b-box {
+  padding: govuk-spacing(4);
+  background-color: govuk-colour("light-grey");
+
+  @include govuk-media-query($until: desktop) {
+    margin-bottom: govuk-spacing(6);
+  }
+}
+
+.app-b-box > * {
```

Comment:
I think we need to avoid styling like this because it can easily style component elements, breaking component isolation (which it does, in this case). I think you can get the same effect by removing this line and adding a `margin_bottom` option to the heading component inside this block.

Reply:
Fixed now.